src/SelectPanel/SelectPanel.tsx

```diff
@@ -0,0 +1,107 @@
+import React, {useCallback, useMemo} from 'react'
+
+import {FilteredActionList, FilteredActionListProps} from '../FilteredActionList'
+import {OverlayProps} from '../Overlay'
+import {ItemInput} from '../ActionList/List'
+import {FocusZoneSettings} from '../behaviors/focusZone'
+import {DropdownButton, DropdownButtonProps} from '../DropdownMenu'
+import {ItemProps} from '../ActionList'
+import {AnchoredOverlay} from '../AnchoredOverlay'
+import Flex from '../Flex'
+
+export interface SelectPanelProps extends Omit<FilteredActionListProps, 'onChange'> {
+  open: boolean
+  onOpen: (gesture: 'anchor-click' | 'anchor-key-press') => unknown
+  onClose: (gesture: 'click-outside' | 'escape') => unknown
+  renderAnchor?: <T extends React.HTMLAttributes<HTMLElement>>(props: T) => JSX.Element
+  placeholder?: string
+  selectedItems: ItemInput[]
```

Comment:
Non-blocking: I wonder if we should merge `items` and `selectedItems`, i.e. add a `selected` boolean to selected items in `items`. Since `items` updates as filters are applied (possible to exclude `selected` items not matching the current filter), we’d need to refactor some logic/assumptions (e.g. perhaps always retaining the full list of `items`, and adding an additional `matchesFilter` boolean).

Reply:
This could get really tricky with combined local/remote loaded items.  I think the current API provides the most flexibility for now, but I'm willing to reconsider as we work on solidifying the API for all the layered components